Frameworks/BlockchainCertificates/Requests/CertificateValidationRequest.swift

```diff
@@ -203,6 +209,19 @@ public class CertificateValidationRequest : CommonRequest {
             do {
                 var json = try JSONSerialization.jsonObject(with: certificate.file, options: []) as! [String: Any]
                 json.removeValue(forKey: "signature")
+                // TODO (kdh): check for added values not mapped to JSON-LD. This is done starting V2
```

Comment:
I opened an issue for that https://github.com/blockchain-certificates/cert-wallet/issues/32

I'll update the TODO